course_dev/ui-HeartView/mac/mainwindow.py

```diff
@@ -435,25 +451,6 @@ def __sendTestRoutine(self):
         # realign plots
         self.autoRangePlots()
 
-    @QtCore.pyqtSlot()
-    def toggleStartStopEnable(self):
-
-        # clear serial to jump to most recent data
-        # Some data may need to be abandonded since it is a real-time system
-        self.ser.clearSerial()
-
-        if (self.start.isEnabled()):
-            self.start.setEnabled(False)
-            self.stop.setEnabled(True)
-
-            self.timerPlotter.start(self.timestep)
-        else:
-            self.start.setEnabled(True)
-            self.stop.setEnabled(False)
-
-            self.timerPlotter.stop()
-
-
```

Comment:
Weird that this function is causing issues. When walking through it everything seems logical but in practice, the toggle (button enable/disable) is inconsistent and often doesn't work correctly. 

To be specific, the `start` button initializes to disabled. The `stop` button initializes to enabled. When pressing the `stop` button then neither button toggles (ie. `start` remains grayed out but can be pressed.. ughhh k.) 

Consider introducing this function in the future as a feature.